mappings/net/minecraft/command/argument/ParticleEffectArgumentType.mapping

```diff
@@ -1,18 +1,28 @@
 CLASS net/minecraft/unmapped/C_wvjakmzf net/minecraft/command/argument/ParticleEffectArgumentType
 	FIELD f_blferatl UNKNOWN_PARTICLE_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
 	FIELD f_dqyoljeq EXAMPLES Ljava/util/Collection;
+	FIELD f_ebwimnfp holderLookup Lnet/minecraft/unmapped/C_vtbxyypo;
+	METHOD <init> (Lnet/minecraft/unmapped/C_bprdujxm;)V
+		ARG 1 context
 	METHOD getExamples getExamples ()Ljava/util/Collection;
 	METHOD listSuggestions listSuggestions (Lcom/mojang/brigadier/context/CommandContext;Lcom/mojang/brigadier/suggestion/SuggestionsBuilder;)Ljava/util/concurrent/CompletableFuture;
 		ARG 1 context
 		ARG 2 builder
+	METHOD m_frlhxnkn (Lcom/mojang/brigadier/StringReader;Lnet/minecraft/unmapped/C_vtbxyypo;)Lnet/minecraft/unmapped/C_qlzoaopu;
+		ARG 0 reader
+		ARG 1 holderLookup
```

Comment:
again, I think it's better to name these holder lookups based on what they're providing than simply holder lookup